pkg/bff/api/v1/api.go

```diff
@@ -25,7 +24,7 @@ type Reply struct {
 
 // StatusReply is returned on status requests. Note that no request is needed.
 type StatusReply struct {
-	Status    string    `json:"status"`
-	Timestamp time.Time `json:"timestamp,omitempty"`
-	Version   string    `json:"version,omitempty"`
+	Status  string `json:"status"`
+	Uptime  string `json:"uptime,omitempty"`
```

Comment:
I prefer uptime to timestamp - it does the same job of busting the cache, but is a little more information which is nice.